src/main/java/school/lemon/changerequest/java/generics/NumberGenerator/SumCalculatorInteger.java

```diff
@@ -0,0 +1,17 @@
+package school.lemon.changerequest.java.generics.NumberGenerator;
+
+/**
+ * Created by lera on 07.01.17.
+ */
+public class SumCalculatorInteger extends SumCalculator<Integer> {
```

Comment:
Name sounds a little bit strange. I'd prefer `IntegerSumCalculator` instead. The same for double calculator.